### DISCUSSION TITLE
a-few-experiments-with-android-drawable-gradients/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-programming#39] [Comments]
# a-few-experiments-with-android-drawable-gradients/

After recently struggling to make a small modification to a simple translucent overlay, I decided to experiment with gradients in Android drawables. After a few hours, I discovered a few new possibilities! Drawable gradients seem to be rarely used despite their simple syntax, with people preferring to use SVGs or static images.

https://blog.jakelee.co.uk/a-few-experiments-with-android-drawable-gradients/

**Comment by hesansadeghi** [comment]
good job,perfect

**Comment by wanwanvxt** [comment]
Thank you! 🤩
